packages/contracts/ignition/modules/CreateOffer.ts

```diff
@@ -0,0 +1,21 @@
+import { buildModule } from "@nomicfoundation/hardhat-ignition/modules";
+
+export default buildModule("CreateOffer", (m) => {
+  const MARKETPLACE_ADDRESS = "0xE49dA88395d76bB4385DfD4d5E0deA19BfDCB3Bd";
+
+  const PRICE_PER_HOUR = 1n;
+  const DEPLOYMENT_FEE = 2n;
+  const FULFILLMENT_TIME = 3n;
+  const UNITS = 4n;
```

Comment:
Let's take all this from a parameters file. You can use the same `parameters/op-sepolia` just add:

```
CreateOffer: {
  MarketplaceAddress: "....",
  PricePerHour.... etc
}
```